build/build.cake

```diff
@@ -207,7 +207,7 @@ Task("Package")
     var buildSettings = new MSBuildSettings {
         MaxCpuCount = 0
     }
-    .SetConfiguration("Release")
+    .SetConfiguration("CI")
     .WithTarget("Pack")
     .WithProperty("GenerateLibraryLayout", "true")
 	.WithProperty("PackageOutputPath", nupkgDir);
```

Comment:
It does that right after the CI build. I think you copied this from the main toolkit repo and forgot to remove the native one. We don't have this configuration at all for the Graph Controls, right? Just remove the build step right after the CI (Line 222 of the build.cake file), and remove it from the solution all together if it's still there.